sources/install/package_mobile.sh

```diff
@@ -6,7 +6,8 @@ source common.sh
 function install_mobile_apt_tools() {
     # CODE-CHECK-WHITELIST=add-aliases
     colorecho "Installing mobile apt tools"
-    fapt android-tools-adb zipalign apksigner apktool scrcpy
+    fapt android-tools-adb zipalign apksigner apktool
+    install_scrpy
```

Comment:
This install function shouldn't be called here. The `install_*_apt_tools()` function is intended for APT install tools like the name implies.